JASP-Engine/JASP/R/common.R

```diff
@@ -2202,6 +2202,18 @@ as.list.footnotes <- function(footnotes) {
   return(NULL)
 }
 
+.suppressGrDevice <- function(plotFunc) {
```

Comment:
@vandenman I figured we could make this a convenience function. I think you also had to do the png trick a few times, right?

Reply:
Well this looks like a fine way to solve it. Another idea is to always open a fake graphics device before an analysis starts and close it again after the analyses is finished (For the rare case that some analysis calls `par` or something like that).